code/online_challenges/src/project_euler/problem_012/problem_012.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+#include <cmath>
+
+int main()
+{
+    int divisorCount = 0;
+    int triangleNumberIndex = 0;
+    int triangleNumber = 0;
+    
+    while (divisorCount < 500)
+    { 
+        divisorCount = 0;
+        ++triangleNumberIndex;
+        triangleNumber += triangleNumberIndex;
+
+        for (int i = 1; i < std::sqrt(triangleNumber) + 1; ++i)
+        {
```

Comment:
Now that this has been shortened so much, we can just remove all of the braces inside this for loop.